docker/JBoss/src/configuration/standalone.xml

```diff
@@ -234,7 +233,7 @@
 				</xa-datasource>
 				<xa-datasource jndi-name="java:/jdbc/ibis4test-mariadb" pool-name="ibis4test-mariadb">
 					<driver>mariadb</driver>
-					<xa-datasource-property name="URL">jdbc:mariadb://${env.jdbc.hostname:host.docker.internal}:3306/testiaf</xa-datasource-property>
+					<xa-datasource-property name="Url">jdbc:mariadb://${env.jdbc.hostname:host.docker.internal}:3306/testiaf</xa-datasource-property>
```

Comment:
- Waarom is de case hier veranderd maar niet bij de MySQL driver? Is dat MariaDB specific?
- Waarom is `pinGlobalTxToPhysicalConnection` hier niet aan de URL toegevoegd? (idem voor Wildfly en WebSphere configs)